src/recipe.rst

```diff
@@ -66,17 +66,20 @@ conda-forge.
 
 Optional: ``bld.bat`` and/or ``build.sh``
 ------------------------------------------
-In many cases, ``bld.bat`` and/or ``build.sh`` files are not required. Pure Python packages almost never need it.
+In many cases, ``bld.bat`` and/or ``build.sh`` files are not required. Pure Python packages almost never need them.
 If the build can be executed with one line, you may put this line in the ``script`` entry of the ``build`` section of
-the ``meta.yaml`` file.
+the ``meta.yaml`` file. (For instance, if a file uses `setuptools`, try ``script: python setup.py install --single-version-externally-managed --record=record.txt``.)
```

Comment:
Do you want to add a note that if you add this line then you should include ``setuptools`` in the build requirements?

Reply:
We should update this to be `script: python -m pip install --no-deps --ignore-installed .` but let's merge this as is and I'll fix that later.